aws/components/directory/setup.ftl

```diff
@@ -257,4 +152,54 @@
         [/#if]
     [/#if]
 
+    [#local epilogue_cidr=[]]
+    [#local epilogue_content=[]]
+
+    [#if ! solution.IPAddressGroups?seq_contains("_global")]
+        [#local epilogue_content += [
+                "   info \"SecGroupId = $\{secgrp_id}\"", 
```

Comment:
You can use the r' syntax which is the equivalent of literal strings 

For example 
https://github.com/hamlet-io/engine-plugin-aws/blob/3c1b1337fb55f8b06d2e7846da7d9936f5986999/aws/extensions/computetask_awslinux_eip/extension.ftl

Reply:
Not sure I agree it adds to the readability as the complex lines have quotes within quotes, so there's no avoiding escaping values, but done